susemanager-utils/susemanager-sls/salt/uptodate.sls

```diff
@@ -31,6 +40,7 @@ mgr_keep_system_up2date_pkgs:
       - sls: channels
       - pkg: mgr_keep_system_up2date_updatestack
 
+{%- if salt['pillar.get']('mgr_reboot_if_needed', True) %}
```

Comment:
Just for understanding. The default value of an unset pillar is `True`? Couldn't find anything on it in the docs

Reply:
Yes, the `True` is the default value if this pillar does not exist
https://docs.saltproject.io/en/latest/ref/modules/all/salt.modules.pillar.html#salt.modules.pillar.get

Reply:
Now I understand. The `True` is the default not the expected value. Thank you